src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -30,6 +34,25 @@
 
 namespace Dynamo.Applications.Models
 {
+    public interface IRevitStartConfiguration : IStartConfiguration
+    {
+        ExternalCommandData ExternalCommandData { get; set; }
+    }
+
+    public struct RevitStartConfiguration : IRevitStartConfiguration
+    {
+        public string Context { get; set; }
+        public string DynamoCorePath { get; set; }
+        public IPreferences Preferences { get; set; }
+        public bool StartInTestMode { get; set; }
+        public IUpdateManager UpdateManager { get; set; }
+        public ISchedulerThread SchedulerThread { get; set; }
+        public string GeometryFactoryPath { get; set; }
+        public IAuthProvider AuthProvider { get; set; }
+        public string PackageManagerAddress { get; set; }
+        public ExternalCommandData ExternalCommandData { get; set; }
+    }
+
```

Comment:
How about we move this to inside of the class like this:

    public class Foo
    {
        public interface IFooStartConfig
        {
            int Bar { get; set; }
        }
    }